src/components/Error.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const ErrorText = styled.Text`
+  font-size: 10;
+  color: red;
+`;
+
```

Comment:
i think better to move all styles components into shared styles folder and export here so i can reuse only styles